userspace/libscap/engine/savefile/CMakeLists.txt

```diff
@@ -4,5 +4,7 @@ add_library(scap_engine_savefile
     scap_reader_gzfile.c
     scap_reader_buffered.c)
 
-add_dependencies(scap_engine_savefile zlib)
+if(NOT MINIMAL_BUILD)
+    add_dependencies(scap_engine_savefile zlib)
```

Comment:
Shouldn't zlib target be always present? https://github.com/falcosecurity/libs/pull/851/files#diff-8bec0e6ac4c47eee0b0d9bf08b8c983788298503fa2624e5e5861d35240adc32R69

Perhaps a better fix is to always include zlib here? https://github.com/falcosecurity/libs/blob/master/userspace/libscap/CMakeLists.txt#L25

Reply:
At least, that was the original purpose of adding the fake targets.

Reply:
I agree, but we currently are inconsistent. We don't have a very predictable way to include dependencies, and we still check for platform and build options when adding dependencies. For now, my attempt was to fix the regression in the build setup to unblock Falco and other consumers, but in the future I would like to work on a refactor and cleanup of the cmake files as a whole so that to remove all the inconsistencies.

Reply:
Then it's good2go!